docs/modules/usage/llms/litellm-proxy.md

```diff
@@ -0,0 +1,55 @@
+
+# LiteLLM Proxy
+
+OpenHands supports using the [LiteLLM proxy](https://docs.litellm.ai/docs/proxy/quick_start) to access various LLM providers. This is particularly useful when you want to:
+
+- Use a single interface to access multiple LLM providers
+- Add authentication, rate limiting, and other features to your LLM API
+- Route requests through a proxy for security or networking requirements
+
+## Configuration
+
+To use LiteLLM proxy with OpenHands, you need to:
+
+1. Set up a LiteLLM proxy server (see [LiteLLM documentation](https://docs.litellm.ai/docs/proxy/quick_start))
+2. Configure OpenHands to use the proxy
+
+Here's an example configuration:
+
+```toml
```

Comment:
This shows how to set it in development workflow. Would it be better to follow the other provider documentations and show how to do it in the UI so documentation is consistent?
If it's too much, I can revise it afterward it has merged. Been meaning to do a consistency check in docs.

Reply:
Yeah sounds good.. can you modify this PR directly though?

Reply:
@xingyaoww as promised, I made it consistent with the rest of the providers. And added to sidebar.